### PR TITLE
Fixes #370 add qualification plan to classification type

### LINKED ISSUE #370: Add QualificationPlan classification type

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

**Comment by msevestre** [comment]
Required to create the structure in PKSim. We should think about making it more generic and defining those constants as string or at least provided a way to add app specific folder and classification

File: src/OSPSuite.Assets/UIConstants.cs [hunk 1]
```diff
@@ -143,6 +143,7 @@ public static class Captions
       public static readonly string ObservedDataFolder = "Observed Data";
       public static readonly string ParameterIdentificationFolder = "Parameter Identifications";
       public static readonly string SensitivityAnalysisFolder = "Sensitivity Analyses";
+      public static readonly string QualificationPlanFolder = "Qualification Plans";
       public static readonly string LLOQ = "LLOQ";
```
Comment (Yuri05) [inline src/OSPSuite.Assets/UIConstants.cs]: I think we will have only 1 Qualification plan per project. but still an open question so far, so let's keep it

File: src/OSPSuite.Core/Domain/ClassificationType.cs [hunk 2]
```diff
@@ -7,6 +7,7 @@ public enum ClassificationType
       Simulation,
       Comparison,
       ParameterIdentification,
-      SensitiviyAnalysis
+      SensitiviyAnalysis,
+      QualificationPlan
```
Comment (Yuri05) [inline src/OSPSuite.Core/Domain/ClassificationType.cs]: Typo: Sensitivi**t**yAnalysis

Comment (msevestre) [reply src/OSPSuite.Core/Domain/ClassificationType.cs]: crap!

Comment (msevestre) [reply src/OSPSuite.Core/Domain/ClassificationType.cs]: well this is old here. I do not want to change it because this would require project conversion

**Pull request merged**